qemu/tests/cfg/fio_windows.cfg

```diff
@@ -0,0 +1,16 @@
+- fio_windows:
+    type = fio_windows
+    only Windows
+    timeout = 300
+    i386, i686:
```

Comment:
use can use below command to install  your application to costumed directory, so that,  you can ignore your guest address bit: 

    msiexec /i "msi path" TARGETDIR="C:\myfolder" /qb

thanks,
Xu